test/specs/scenarios/documentSelector.js

```diff
@@ -0,0 +1,19 @@
+import { describe, it } from '../../utils/mochaw'
+import { localhostUrl } from '../../utils/config'
+
+export const documentSelectorScenarios = (driver, screens, lang) => {
+  const { welcome, documentSelector, common } = screens
```

Comment:
In some files, we have these `const`s in one line in others one below another. Shall we keep the formatting consistent?

Reply:
Yes, I will make it consistent

Reply:
I agree with @kopijunkie, I think I misread the original comment. I thought you meant that there was an inconsistence in where these variables were being defined within the files.